scout/constants/so_terms.py

```diff
@@ -15,30 +15,33 @@
     "transcript_amplification": {"rank": 13, "region": "exonic"},
     "regulatory_region_ablation": {"rank": 14, "region": "regulatory_region"},
     "splice_region_variant": {"rank": 15, "region": "splicing"},
-    "incomplete_terminal_codon_variant": {"rank": 16, "region": "exonic"},
-    "synonymous_variant": {"rank": 17, "region": "exonic"},
-    "start_retained_variant": {"rank": 18, "region": "exonic"},
-    "stop_retained_variant": {"rank": 19, "region": "exonic"},
-    "coding_sequence_variant": {"rank": 20, "region": "exonic"},
-    "mature_miRNA_variant": {"rank": 21, "region": "ncRNA_exonic"},
-    "5_prime_UTR_variant": {"rank": 22, "region": "5UTR"},
-    "3_prime_UTR_variant": {"rank": 23, "region": "3UTR"},
-    "non_coding_transcript_exon_variant": {"rank": 24, "region": "ncRNA_exonic"},
-    "non_coding_exon_variant": {"rank": 25, "region": "ncRNA_exonic"},
-    "non_coding_transcript_variant": {"rank": 26, "region": "ncRNA_exonic"},
-    "nc_transcript_variant": {"rank": 27, "region": "ncRNA_exonic"},
-    "intron_variant": {"rank": 28, "region": "intronic"},
-    "NMD_transcript_variant": {"rank": 29, "region": "ncRNA"},
-    "upstream_gene_variant": {"rank": 30, "region": "upstream"},
-    "downstream_gene_variant": {"rank": 31, "region": "downstream"},
-    "TFBS_ablation": {"rank": 32, "region": "TFBS"},
-    "TFBS_amplification": {"rank": 33, "region": "TFBS"},
-    "TF_binding_site_variant": {"rank": 34, "region": "TFBS"},
-    "regulatory_region_amplification": {"rank": 35, "region": "regulatory_region"},
-    "regulatory_region_variant": {"rank": 36, "region": "regulatory_region"},
-    "feature_elongation": {"rank": 37, "region": "genomic_feature"},
-    "feature_truncation": {"rank": 38, "region": "genomic_feature"},
-    "intergenic_variant": {"rank": 39, "region": "intergenic_variant"},
+    "splice_donor_5th_base_variant": {"rank": 16, "region": "splicing"},
+    "splice_donor_region_variant": {"rank": 17, "region": "splicing"},
+    "splice_polypyrimidine_tract_variant": {"rank": 18, "region": "splicing"},
```

Comment:
Looks good to me. Any reason why we should merge it as it is @dnil?

Reply:
I can check, but I dont think so! It was a draft when I looked last. 😊

Reply:
Ah, and there was the idea to parse directly from the full SO or retrieve from VEP, but that is likely overkill.

Reply:
Do you have a case with those terms mixed in @northwestwitch for testing?

Reply:
Otw the regular tests pass, and this shoould work..

Reply:
> Do you have a case with those terms mixed in @northwestwitch for testing?

Nope, unfortunately

Reply:
> Otw the regular tests pass, and this shoould work..

Agreed!